src/csharp/Microsoft.Spark.Worker.UnitTest/PayloadReader.cs

```diff
@@ -60,5 +60,11 @@ public static List<object[]> Read(Stream inputStream)
             
             return rowsReceived;
         }
+
+        public static int ReadInt(Stream inputStream)
```

Comment:
Why is this method needed?  Can't the caller use `SerDe.ReadInt32` directly instead ?

Reply:
I just kept it for sake of consistency.. removed.